### DISCUSSION TITLE
Cross Attention Map

### DISCUSSION THREAD

**Discussion by miriam-horovicz-ni** [huggingface/diffusers#6792] [Q&A] [answered]
I am using Image to Image inpaint pipeline, I want to visualize my generated image cross attention maps in regards to prompt. What is the easiest way to achieve that?

**Comment by sayakpaul** [comment, marked as answer]
You can refer to the https://github.com/yuval-alaluf/Attend-and-Excite codebase and repurpose it accordingly.
